technical/taster-classic-badge-integration.md

```diff
@@ -0,0 +1,72 @@
+# Taster Classic Badge Integration
+
+This document is intended for those who want to integrate the classic badge into their pilot. Please follow these instructions carefully, and if you wish to deviate from the default settings for any reason, please discuss this with a member of the Taster team (tastertech@rd.bbc.co.uk).
+
+There are two types of classic quickrate badge:
+
+| `<panel_type>` | use |
+| --------------|-----|
+| `link` | The panel features a button that, when clicked, opens the appropriate Taster Pilot Page in a new window, with the rating dropdown open |
+| `rate` | Keeps users on your page without directing them back to Taster. For users without JavaScript this behaves like the link panel. With this widget the users can rate and answer survey questions on the page and the contents of the iframe update themselves accordingly |
+
+## Integration Guide
+
+There are several components and specifications that must be included in your site to allow the slim badge to operate correctly. They should be included on any page that is visible to the user.
+
+1. The head of the document should contain the following script tag:
+
+  ```
+  <script type="text/javascript" src="http://www.bbc.co.uk/taster/pilot-lib-slim.js"></script>
+  ```
+
+2. The iframe (which contains the quickrate badge) should be placed on the page with the following lines which initiate an iFrame pointed at the slim badge for your pilot. Please note that `<pilot-id>` should be replaced with the pilot ID that you have been assigned and `<panel_type>` should be relaced with `link` or `rate` according to the desired state described in the table above (if you are uncertain of these settings, please speak to the Taster team or refer to the Testing section at the end of this document):
+
+  ```
+  <iframe class="taster-offsite-panel" src="http://www.bbc.co.uk/taster/projects/<pilot-id>/offsite/slim" frameborder="0" scrolling="no"></iframe>
+  ```
+
+3. At the end of the HTML document, the following code should be entered:
+
+  ```
+  <script type="text/javascript">
+      require(['pilot-lib-slim/taster-offsite-panel'], function (Panel) {
+          var panel = new Panel('.taster-offsite-panel');
+      });
+  </script>
+  ```
+
+4. In order for the badge to operate correctly, certain CSS rules need to applied to the iframe as follows:
+
+  ```
+  .taster-offsite-panel {
+    position: fixed;
+    right: 0;
+    top: 0;
+    width: 56px;
+    z-index: 2;
+  }
+  ```
```

Comment:
That entire section is unedited slim badge text. I think it might have been pending decisions around the 'wide' badge. I'll update to assume it is using the classic badge for just now and we can shift to the 'wide' badge later.